js/src/collections/common/scores.js

```diff
@@ -3,9 +3,24 @@ var Handlebars = require('handlebars');
 
 var CONFIG = require('../../../config.json');
 
+var ScoresByIndicatorSQL = Handlebars.compile(require('../../queries/scores/scores_by_target.hbs'));
+
 var ScoresCollection = CartoDBCollection.extend({
 
-  scores_table: CONFIG.cartodb.scores_table
+  scores_table: CONFIG.cartodb.scores_table,
+  indicators_table: CONFIG.cartodb.indicators_table,
+
+  getScoresByTarget: function(settings) {
```

Comment:
If you are getting here the scores for one indicator, maybe there would be a better name.